azure-pipelines.prod.yml

```diff
@@ -0,0 +1,47 @@
+trigger: 
+  - master
+  - releases/*
+pr: none
+
+pool:
+   vmImage: 'vs2017-win2016'
+
+steps:
+   # Azure Key Vault
+   # Download Azure Key Vault secrets
+   - task: AzureKeyVault@1
+     inputs:
+       azureSubscription: 'hazzat-api - Azure'
+       keyVaultName: 'hazzat-api'
+       secretsFilter: 'hazzatdnn-ConnectionString'
+
+   - task: NodeTool@0
+     inputs:
+       versionSpec: '10.x'
+     displayName: 'Install Node.js'
+
+   - task: Npm@1
+     displayName: 'npm install'
+     inputs:
+       command: custom
+       workingDir: src
+       verbose: false
+       customCommand: 'install'
+
+   - task: Npm@1
+     displayName: 'npm build'
+     inputs:
+       command: custom
+       workingDir: src
+       verbose: false
+       customCommand: 'run build'
+
+   - task: Npm@1
+     displayName: 'npm test'
+     inputs:
+       command: custom
+       workingDir: src
+       verbose: false
+       customCommand: 'run test'
+     env:
+       CUSTOMCONNSTR_DBCONNECTIONSTRING: $(hazzatdnn-ConnectionString)
```

Comment:
Prod pipeline should also publish the zip file just like develop

Reply:
That's the part I need your help with, my current permissions don't allow me to create new resource targets.

Reply:
done